develop/plone/sessions/cookies.rst

```diff
@@ -60,15 +60,15 @@ Clearing cookies
 
 If a cookie needs to be removed, the browser has to be told to expire it.
 The ``expireCookie`` method does this.
-It always set ``max_age`` to ``0`` and the ``expires`` date to the past.
+It always sets ``max_age`` to ``0`` and the ``expires`` date to the past.
 Additional the cookie value will be set to ``deleted``.
 
 .. code-block:: python
 
     self.request.response.expireCookie("cookie_name")
 
 The method allows additional attributes to be passed as kwargs, similar to ``setCookie``.
-Except ``max_age``, ``expires`` and ``value`` all is allowed, like `path` or other cookie attributes.
+Except ``max_age``, ``expires`` and ``value`` all is allowed, like ``path`` or other cookie attributes.
```

Comment:
These two sentences together on lines 70-71 still don't make sense to me. Can you clarify?

Reply:
You can set any additional attribute to the expiring cookie, except these 3.